### ISSUE TITLE
The plugin doesn't pick up relation between field `list`and it's usage  

### ISSUE THREAD

**Issue by grisevg** [TiVo/intellij-haxe#234]
Plugin version: 0.9.3
IDEA: 14.1.1
OS: Win 8.1 x64
The plugin thinks that it's `List` class instead:
```haxe
class Foo {
 var list:Int;
 public function new() {
   list = 5;
 }
}
```

TIR: STB-9403

**Comment by EBatTiVo** [comment]
This is related to #95.

**Comment by as3boyan** [comment]
This is ok:

```haxe
class Foo {
    var list2:Int;
    public function new() {
        list2 = 5;
    }
}
```

So the thing is with resolving list, it thinks that it is a class.

**Comment by EBatTiVo** [comment]
Fixed with #406: https://github.com/TiVo/intellij-haxe/commit/d44634bceb2c799e77714ddbdea44818284b1c1d
